a.core_concepts.md

```diff
@@ -158,13 +158,13 @@ kubectl create quota myrq --hard=cpu=1,memory=1G,pods=2 --dry-run -o yaml
 <p>
 
 ```bash
-kubectl get po --all-namespaces
+kubectl get po -A # or the more verbose --all-namespaces
 ```
 
 </p>
 </details>
 
-### Create a pod with image nginx called nginx and allow traffic on port 80
+### Create a pod with image nginx called nginx, expose traffic to the container on port 80 without creating a service
```

Comment:
'allow' could be interpreted as a network policy, expose is what actually happens but that would be too suggestive and might trick people into using --expose?

Reply:
Agreed, good point